tests/globalhelper/reporthelper.go

```diff
@@ -188,3 +188,12 @@ func removeCharactersFromString(stringToFormat string, charactersToRemove []stri
 func formatTestCaseName(tcName string) string {
 	return removeCharactersFromString(tcName, []string{"-", "_", " "})
 }
+
+func ConvertSpecNameToFileName(specName string) string {
```

Comment:
Can you rewrite this function using for loop?
for _, symbol := range []string(" ",", ","-") {
....
}